ui/app/AppLayouts/Wallet/views/LeftTabView.qml

```diff
@@ -104,65 +128,124 @@ Rectangle {
         }
 
         StatusListView {
+            id: walletAccountsListView
+
             objectName: "walletAccountsListView"
             spacing: Style.current.smallPadding
             Layout.fillWidth: true
             Layout.fillHeight: true
             Layout.topMargin: Style.current.halfPadding
+            Layout.leftMargin: -Style.current.padding
+            Layout.rightMargin: -Style.current.padding
```

Comment:
usually not a good practice to use negative margins unless it's inevitable...wouldn't just removing `        anchors.margins: Style.current.padding` from ColumnLayout work?